pkg/service/model_builder.go

```diff
@@ -63,6 +71,20 @@ func (b *defaultModelBuilder) Build(ctx context.Context, service *corev1.Service
 		defaultHealthCheckHealthyThreshold:   3,
 		defaultHealthCheckUnhealthyThreshold: 3,
 	}
+	targetType, err := task.buildTargetType(ctx)
+	if err != nil {
+		return nil, nil, err
+	}
+	if targetType == elbv2model.TargetTypeInstance &&
```

Comment:
tbh, i think this is a little bit hacky here. model_builder's build should be simply and delegate to task run directly(instead of have different default values here).

instead, i think it's better to split these default values into different variables.
e.g. `defaultHealthCheckintervalForIPMode` and `defaultHealthCheckIntervalForLocalInstanceMode`.

Reply:
I was trying to avoid more code. I will decouple the default values and update the PR.